cmd/root.go

```diff
@@ -143,62 +198,57 @@ any client SSL certificates.`,
 
 	cmd.RunE = func(*cobra.Command, []string) error { return runSignalWrapper(c) }
 
+	// Override Cobra's default messages.
+	cmd.PersistentFlags().BoolP("help", "h", false, "Display help information for cloudsql-proxy")
+	cmd.PersistentFlags().BoolP("version", "v", false, "Print the cloudsql-proxy version")
+
 	// Global-only flags
 	cmd.PersistentFlags().StringVarP(&c.conf.Token, "token", "t", "",
-		"Bearer token used for authorization.")
+		"Use bearer token as a source of IAM credentials.")
 	cmd.PersistentFlags().StringVarP(&c.conf.CredentialsFile, "credentials-file", "c", "",
-		"Path to a service account key to use for authentication.")
+		"Use service account key file as a source of IAM credentials.")
 	cmd.PersistentFlags().BoolVarP(&c.conf.GcloudAuth, "gcloud-auth", "g", false,
-		"Use gcloud's user configuration to retrieve a token for authentication.")
+		"Use gcloud's user credentials as a source of IAM credentials.")
 	cmd.PersistentFlags().BoolVarP(&c.conf.StructuredLogs, "structured-logs", "l", false,
-		"Enable structured logs using the LogEntry format")
+		"Enable structured logging with LogEntry format")
 	cmd.PersistentFlags().Uint64Var(&c.conf.MaxConnections, "max-connections", 0,
-		`Limits the number of connections by refusing any additional connections.
-When this flag is not set, there is no limit.`)
+		"Limit the number of connections. Default is no limit.")
 	cmd.PersistentFlags().DurationVar(&c.conf.WaitOnClose, "max-sigterm-delay", 0,
-		`Maximum amount of time to wait after for any open connections
-to close after receiving a TERM signal. The proxy will shut
-down when the number of open connections reaches 0 or when
-the maximum time has passed. Defaults to 0s.`)
+		"Maximum time to wait for connections to close after receiving a TERM signal.")
 	cmd.PersistentFlags().StringVar(&c.telemetryProject, "telemetry-project", "",
-		"Enable Cloud Monitoring and Cloud Trace integration with the provided project ID.")
+		"Enable Cloud Monitoring and Cloud Trace with the provided project ID.")
 	cmd.PersistentFlags().BoolVar(&c.disableTraces, "disable-traces", false,
```

Comment:
It would probably be clearer if we make the telemetry `traces` and `metrics` semantically positive default true rather than negative. (This would need to be in a different PR, obviously)

```
--telemetry-project=name   # project=name, traces on, metrics on
--telemetry-project=name --enable-traces=false  #project=name, traces off, metrics on
--telemetry-project=name --enable-metrics=false  #project=name, traces on, metrics off
```

Reply:
It's a question of whether `--enable-metrics=false` reads better than `--disable-metrics`. I find the `enable` and `false` words to require a careful reading.

```
--telemetry-project=name   # project=name, traces on, metrics on
--telemetry-project=name --disable-traces  #project=name, traces off, metrics on
--telemetry-project=name --disable-metrics  #project=name, traces on, metrics off
```

Reply:
Oh, yeah. I see. This is good enough for now. I think we can improve on it 

This is how it is handled in  [`curl` man page](https://man7.org/linux/man-pages/man1/curl.1.html) 

```
       In general, all boolean options are enabled with --option and yet
       again disabled with --no-option. That is, you use the exact same
       option name but prefix it with "no-". However, in this list we
       mostly only list and show the --option version of them. (This
       concept with --no options was added in 7.19.0. Previously most
       options were toggled on/off through repeated use of the same
       command line option.)
```